### PR TITLE
CON-3133 Fix ``AI/emotions detector``

### PR DESCRIPTION
**Pull request by Oumaimafisaoui**
Resolves: CON-3133

- Update train.py to save learning curves and model architecture
- Modify predict.py to evaluate on test set instead of validation set
- Ensure 62% accuracy requirement on test set

### CODE REVIEW

File: subjects/ai/emotions-detector/README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 - emotion classification
 - face tracking
 
 With the computing power exponentially increasing the computer vision field has been developing exponentially. This is a key element because the computer power allows using more easily a type of neural networks very powerful on images:
-CNN's (Convolutional Neural Networks). Before the CNNs were democratized, the algorithms used relied a lot on human analysis to extract features which obviously time-consuming and not reliable. If you're interested in the "old
-school methodology" [this article](https://towardsdatascience.com/classifying-facial-emotions-via-machine-learning-5aac111932d3) explains it.
-The history behind this field is fascinating! [Here](https://kapernikov.com/basic-introduction-to-computer-vision/) is a short summary of its history.
+
+- CNN's (Convolutional Neural Networks). Before the CNNs were democratized, the algorithms used relied a lot on human analysis to extract features which obviously time-consuming and not reliable. If you're interested in the "old
+  school methodology" [this article](https://towardsdatascience.com/classifying-facial-emotions-via-machine-learning-5aac111932d3) explains it.
```
Comment (MSilva95) [inline subjects/ai/emotions-detector/README.md:18]: This should be i the same line, it looks odd in the preview

**Review by MSilva95 — CHANGES_REQUESTED** [review]
just a small change

File: subjects/ai/emotions-detector/README.md (hunk 1 above)
Comment (Oumaimafisaoui) [reply subjects/ai/emotions-detector/README.md]: It was caused by ``prettier``, but I fixed it !

**Review by MSilva95 — APPROVED** [review]
LGTM!
